README.md

```diff
@@ -18,7 +18,6 @@ Check out the [Getting Started](http://facebook.github.io/jest/docs/getting-star
 <generated_api_start />
 #### The `jest` object
 
-  - `jest.addMatchers(matchers)`
```

Comment:
ah, the README is updated by the publish script. This was already removed in `API.md` in a previous commit but I didn't push the website.